build.gradle

```diff
@@ -1491,13 +1491,14 @@ project(':streams') {
   }
 
   tasks.create(name: "copyDependantLibs", type: Copy) {
-    from (configurations.testRuntime) {
```

Comment:
@vvcephei do you know why we're copying test dependencies here?

Reply:
Hint/note: this **_'from'_** clause could be redundant now:

![image](https://user-images.githubusercontent.com/19467899/116138154-bd0dae00-a6d4-11eb-8017-5fdde99239f9.png)

Reply:
One more note: @vvcephei didn't introduced/changed `copyDependentLibs` Gradle task. I just mentioned him above while I was investigating changes related to a `streams` sub-module dependencies block (at the time when I just started to look for an approach). 

That specific Gradle task was introduced as a part of this (big) improvement: https://cwiki.apache.org/confluence/display/KAFKA/KIP-28+-+Add+a+processor+client
Related (pretty large) commit: https://github.com/apache/kafka/commit/263c10ab7c8e8fde9d3566bf59dccaa454ee2605

Reply:
So... I will test full Gradle build (across all Scala versions and with all tests) with and without this clause. 
If turns out that build results are comparable I will remove that clause.

Reply:
That line should be including test dependencies like log4j, etc. But I don't know why that makes sense for streams. It does make sense for core. So, I would remove it.

Reply:
Ok, we have a deal then.